src/app/item-page/simple/item-page.component.ts

```diff
@@ -57,13 +62,38 @@ export class ItemPageComponent implements OnInit {
 
   itemUrl: string;
 
+  /**
+   * Contains a list of SignpostingLink related to the item
+   */
+  signpostingLinks: SignpostingLink[] = [];
+
   constructor(
     protected route: ActivatedRoute,
-    private router: Router,
-    private items: ItemDataService,
-    private authService: AuthService,
-    private authorizationService: AuthorizationDataService
+    protected router: Router,
+    protected items: ItemDataService,
+    protected authService: AuthService,
+    protected authorizationService: AuthorizationDataService,
+    protected responseService: ServerResponseService,
+    protected signpostingDataService: SignpostingDataService,
+    protected linkHeadService: LinkHeadService
   ) {
+    this.route.params.subscribe(params => {
+      this.signpostingDataService.getLinks(params.id).pipe(take(1)).subscribe((signpostingLinks: SignpostingLink[]) => {
+        let links = '';
+        this.signpostingLinks = signpostingLinks;
+
+        signpostingLinks.forEach((link: SignpostingLink) => {
+          links = links + (isNotEmpty(links) ? ', ' : '') + `<${link.href}> ; rel="${link.rel}" ; type="${link.type}" `;
+          this.linkHeadService.addTag({
+            href: link.href,
+            type: link.type,
+            rel: link.rel
+          });
+        });
+
+        this.responseService.setHeader('Link', links);
```

Comment:
We might want to split this into a different method so that we can describe the logic here.  It's not at all straightforward.... but it appears to be generating the proper `Link` headers to send back?  

Is there any way to add specs for this logic?  I noticed they don't currently exist (unless I'm overlooking them)